main.go

```diff
@@ -30,10 +30,11 @@ import (
 	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
 	"k8s.io/kubernetes/cmd/cloud-controller-manager/app/options"
 	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
-	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
-	_ "k8s.io/kubernetes/pkg/version/prometheus" // for version metric registration
+	_ "k8s.io/kubernetes/pkg/version/prometheus"        // for version metric registration
 	"k8s.io/kubernetes/pkg/version/verflag"
 
+	"github.com/dims/openstack-cloud-controller-manager/pkg/openstack"
```

Comment:
should be `git.openstack.org/openstack/openstack-cloud-controller-manager`

please see https://github.com/dims/openstack-cloud-controller-manager/blob/master/glide.yaml#L1

Reply:
OK, But it is a litter strange like this.

Reply:
It is better to update glide.yaml too.

Reply:
@edisonxiang the plan is to move to openstack git repo when we have the basic functionality working hence the request to use the openstack git url in the packaging